### ISSUE TITLE
Linking Standard and Conversational Forms: Options and Possibilities

### ISSUE THREAD

**Issue by Omineus** [fluentform/fluentform#509]
I want to create two forms that should be linked:

A standard form for order and payment.
A conversational form to gather information about the order.
Is it possible to link certain elements, such as specific questions from the standard form, to the conversational form?

Alternatively, can we generate a unique link at the end of the standard (payment) form that redirects to a customized conversational form? This way, the link would be unique and tailored to the conversational form.

**Comment by Omineus** [comment]
We have a service form where we also discuss the business part. Is it possible to split it into two different forms and then have them sort of merged together?

For example:
The customer fills out the form 1 with all services checked, such as: Product design, along with two additional white photos.

And at the end, they pay the invoice, can they then receive a customized link to the service form 2 to provide the information, which also has linked information from form 1?

**Comment by nkb-bd** [comment]
I am afraid currently we do not have this, but We have custom development team, if you want I can forward you to them.

**Comment by Omineus** [comment]
Yes please forward it. What does that mean? Will they make it for public?
